Mention that calling some_function.remote() is non-blocking (#7417)
ray-project/ray: doc/source/walkthrough.rst

       for _ in range(4):
           remote_function.remote()

The invocations are executed in parallel because the call to ``remote_function.remote()`` doesn't block.

> All I/O is performed in the background using Ray's internal event loop.

Reviewer: ```suggestion
All computation is performed in the background, driven by Ray's internal event loop.
```
